docs/ecosystem/roadmap.md

```diff
@@ -71,30 +71,25 @@ All collected money goes directly into open source development.
 
 ### Changelogs
 
-- [Ory Kratos Changelog](https://github.com/ory/kratos/blob/master/CHANGELOG.md) /
-  [Releases](https://github.com/ory/kratos/releases)
-- [Ory Hydra Changelog](https://github.com/ory/hydra/blob/master/CHANGELOG.md) / [Releases](https://github.com/ory/hydra/releases)
-- [Ory Keto Changelog](https://github.com/ory/keto/blob/master/CHANGELOG.md) / [Releases](https://github.com/ory/keto/releases)
-- [Ory Oathkeeper Changelog](https://github.com/ory/oathkeeper/blob/master/CHANGELOG.md) /
-  [Releases](https://github.com/ory/oathkeeper/releases)
+[Visit the Changelog for all Ory Changelogs](./changelog.md)
 
 ### Milestones, issues, pull requests
 
 Ory Kratos: [Milestones](https://github.com/Ory/kratos/milestones) /
-[Open Issues & Pull Requests](https://github.com/ory/kratos/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc) /
-[Issues sorted by popularity](https://github.com/ory/kratos/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc)
+[Open Issues & Pull Requests](https://github.com/ory/kratos/issues?q=is%3Aopen) /
+[Issues sorted by popularity](https://github.com/ory/kratos/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions)
 
 Ory Hydra: [Milestones](https://github.com/Ory/hydra/milestones) /
-[Open Issues & Pull Requests](https://github.com/ory/hydra/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc) /
-[Issues sorted by popularity](https://github.com/ory/hydra/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc)
+[Open Issues & Pull Requests](https://github.com/ory/hydra/issues?q=is%3Aopen) /
+[Issues sorted by popularity](https://github.com/ory/hydra/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions)
 
 Ory Keto: [Milestones](https://github.com/ory/keto/milestones) /
-[Open Issues & Pull Requests](https://github.com/ory/keto/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc) /
-[Issues sorted by popularity](https://github.com/ory/keto/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc)
+[Open Issues & Pull Requests](https://github.com/ory/keto/issues?q=is%3Aopen) /
+[Issues sorted by popularity](https://github.com/ory/keto/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions)
 
 Ory Oathkeeper: [Milestones](https://github.com/ory/oathkeeper/milestones) /
-[Open Issues & Pull Requests](https://github.com/ory/oathkeeper/issues?q=is%3Aopen+sort%3Aupdated-desc+sort%3Areactions-%2B1-desc)
-/ [Issues sorted by popularity](https://github.com/ory/oathkeeper/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc)
+[Open Issues & Pull Requests](https://github.com/ory/oathkeeper/issues?q=is%3Aopen) /
+[Issues sorted by popularity](https://github.com/ory/oathkeeper/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions)
```

Comment:
I suggest merging this into the roadmap chapter on line 14. Milestones are definitely a part of the roadmap. So are issues (where we discuss items that end up on the roadmap) and pull requests (that implement parts of the roadmap). 

I'm not sure it's helpful to see both open issues and pull requests at the same time. These are two very different things, the fact that Github implemented PRs as issues is a quirky implementation detail. I suggest providing separate links to each category.